release.yaml

```diff
@@ -1,286 +1,65 @@
 fleet:
-- 100.0.4+up0.3.10-rc5
-- 100.0.0+up0.3.6
-- 100.0.1+up0.3.7
-- 100.0.2+up0.3.8
-- 100.0.3+up0.3.9
-- 0.3.900+up0.3.9
 - 0.3.900+up0.3.9
-fleet-agent:
 - 100.0.4+up0.3.10-rc5
-- 100.0.0+up0.3.6
-- 100.0.1+up0.3.7
-- 100.0.2+up0.3.8
-- 100.0.3+up0.3.9
-- 0.3.900+up0.3.9
+fleet-agent:
 - 0.3.900+up0.3.9
-fleet-crd:
 - 100.0.4+up0.3.10-rc5
-- 100.0.0+up0.3.6
-- 100.0.1+up0.3.7
-- 100.0.2+up0.3.8
-- 100.0.3+up0.3.9
-- 0.3.900+up0.3.9
+fleet-crd:
 - 0.3.900+up0.3.9
-harvester-cloud-provider:
-- 100.0.0+up0.1.8
-- 100.0.1+up0.1.8
-- 100.0.2+up0.1.12
+- 100.0.4+up0.3.10-rc5
 harvester-csi-driver:
 - 100.0.3+up0.1.14
-- 100.0.0+up0.1.9
```

Comment:
Cleaned up all these things from the release.yaml that were already merged as part of an OOB release